.github/workflows/test.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         with:
           name: cronos
           extraPullNames: dapp
+          installCommand: "nix-env --quiet -j8 -iA cachix -f https://cachix.org/api/v1/install/1.6.1"
```

Comment:
What's the difference of this specific version?

Reply:
seems will have a [fix](https://github.com/cachix/cachix-action/issues/168#issuecomment-1900086943) soon